applications/serial_lte_modem/doc/HTTPC_AT_commands.rst

```diff
@@ -128,9 +128,10 @@ Syntax
   Each header field should end with ``<CR><LF>``.
 * The ``<payload_length>`` is an integer.
   It represents the length of the payload.
-  If ``payload_length`` is greater than ``0``, the SLM will enter the pass-through mode and expect the upcoming UART input data as payload.
+  If ``payload_length`` is greater than ``0``, the SLM will enter the data mode and expect the upcoming UART input data as payload.
   The SLM will then send the payload to the HTTP server until the ``payload_length`` bytes are sent.
-  To abort sending the payload, use ``AT#XHTTPCCON=0`` to disconnect from the server.
+  To abort sending the payload, use ``CONFIG_SLM_DATAMODE_TERMINATOR`` to terminate data mode. The default pattern string is "+++".
+  UART silence ``CONFIG_SLM_DATAMODE_SILENCE`` before and after the pattern string is used to exit data mode.
```

Comment:
```suggestion
  To abort sending the payload, terminate data mode by sending the terminator string defined in :option:`CONFIG_SLM_DATAMODE_TERMINATOR`.
  The default pattern string is "+++".
  Keep in mind that UART silence as configured in :option:`CONFIG_SLM_DATAMODE_SILENCE` is required before and after the pattern string.
```

Reply:
The description was changed per suggestion.